### PR TITLE
Fish 7957 fix webservices jacc tck

### PR DESCRIPTION
**Pull request by aubi**
Fixes webservices and jacc TCKs
Includes James' change of handling derby files

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -72,10 +83,6 @@ if [ -z "$SKIP_TCK" ]; then
       echo "Replacing runner script with patched one"
       cp patch/run_jakartaeetck.sh $WORKSPACE/docker/
     fi;
-
-    # Copy patched ant_sun.jar
-    echo "Replacing ant_sun.jar with patched one"
-    cp patch/ant_sun.jar $CTS_HOME/jakartaeetck/lib/ant_sun.jar
```
Comment (Pandrex247) [inline run.sh:78]: Why are you removing this? This is important
Without this we fail the TCK if the Payara version contains "8"

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

File: run.sh (hunk 1 above)
Comment (Pandrex247) [reply run.sh:78]: See:
* https://github.com/payara/jakartaeetck-runner/pull/85
* https://github.com/payara/patched-src-jakartaee-tck-tools/pull/1/files

**Comment by aubi** [comment]
Fixed, running new build now.

**Review by breakponchito — APPROVED** [review]
LGTM

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**